client/landing/stepper/declarative-flow/internals/types.ts

```diff
@@ -46,6 +46,10 @@ export type DeprecatedStepperStep = {
 	 * The step slug is what appears as part of the pathname. Eg the intro in /setup/link-in-bio/intro
 	 */
 	slug: string;
+	/**
+	 * Does the step require a logged-in user?
+	 */
+	requiresLoggedInUser?: boolean;
```

Comment:
We should make a new type entirely that requires (`requiresLoggedInUser`) and deprecate the existing ones, so people get deprecation warnings from TS and the flows would organically upgrade to the new type.

I do something like that [here](https://github.com/Automattic/wp-calypso/blob/3a94454b152d4a683c7c69582d212f59d0d91488/client/landing/stepper/declarative-flow/internals/types.ts#L70).

Reply:
You're right. Maybe it should be `requiresLoggedInUser?: true`?